source/javascripts/modules/platform_placeholder.js

```diff
@@ -0,0 +1,33 @@
+class platformPlaceholder {
```

Comment:
@skatkov I must admit that I've never written unit tests for JS logic before and would love to find the best way to do it, but I'm afraid it will take more time than we can afford right now :'(